src/controllers/SubscriberController.ts

```diff
@@ -242,4 +241,13 @@ export class SubscriberController {
         const callBack = this.config.onShapeCornerRadiusChanged;
         callBack && callBack(JSON.parse(cornerRadius));
     };
+
+    /**
+     * Listener of editor entering / exiting the crop mode
+     * @param id frame id when entering / null when exiting
+     */
+    onCropActiveFrameIdChanged = (id?: Id) => {
```

Comment:
Not really a request but just a thought:

The API is nice to me internally but i'm wondering integrator-ux-wise if mapping to:

`onCropModeEnter = (id: Id)` and `onCropActiveFrameExit = ()` would fit better.

Reply:
Had a call w/ @evan-mcgeek about that, this is obviously an open question but:
- 2 different events for this might not make the best sense
- With this change we would have 3 crop-related streams